src/cbc_sdk/platform/vulnerability_assessment.py

```diff
@@ -320,6 +401,22 @@ def add_criteria(self, key, value, operator='EQUALS'):
         self._update_criteria(key, value, operator)
         return self
 
+    def set_deployment_type(self, deployment_type, operator):
+        """
+        Restricts the vulnerabilities that this query is performed on to the specified deployment type.
+
+        Args:
+            deployment_type (str): deployment type ("ENDPOINT", "AWS")
+            operator (str):  logic operator to apply to property value.
+
+        Returns:
+            VulnerabilityQuery: This instance.
+        """
+        if not deployment_type:
+            raise ApiError("Invalid deployment type")
+        self._update_criteria("deployment_type", deployment_type, operator)
```

Comment:
Should we have some validation on deployment_types or is not needed?

Reply:
I'm hesitant to add this with all the public cloud items coming out. The API will respond with an error anyways so it only saves us an API call